src/config/env/dev.json

```diff
@@ -1,28 +1,26 @@
 {
-  "CHAIN_ID": "3",
+  "CHAIN_ID": "5",
   "LOCAL_STORAGE_KEY": "builder",
   "EMAIL_SERVER_URL": "https://decentraland.org/subscribe",
   "PEER_URL": "https://peer.decentraland.zone",
   "ERC721_COLLECTION_BASE_URI": "https://peer.decentraland.zone/lambdas/collections/standard/erc721/",
   "BUILDER_SERVER_URL": "https://builder-api.decentraland.io/v1",
   "MARKETPLACE_URL": "https://api.decentraland.io/v1",
   "MARKETPLACE_WEB_URL": "https://market.decentraland.io",
-  "LAND_MANAGER_URL": "https://api.thegraph.com/subgraphs/name/decentraland/land-manager-ropsten",
+  "LAND_MANAGER_URL": "https://api.thegraph.com/subgraphs/name/decentraland/land-manager-goerli",
   "SHARE_SCENE_URL": "https://share.decentraland.zone/b",
   "INTERCOM_APP_ID": "z0h94kay",
   "SEGMENT_API_KEY": "H21EgRI4eYwICDZf5uW6ek2BiykIR6wA",
   "ROLLBAR_ACCESS_TOKEN": "46e4b7e45c844b9ab81315c8b0919e99",
   "ENVIRONMENT": "development",
-  "MANA_TOKEN_CONTRACT_ADDRESS": "0x2a8fd99c19271f4f04b1b7b9c4f7cf264b626edb",
-  "LAND_REGISTRY_CONTRACT_ADDRESS": "0x7a73483784ab79257bb11b96fd62a2c3ae4fb75b",
-  "ESTATE_REGISTRY_CONTRACT_ADDRESS": "0x124bf28a423b2ca80b3846c3aa0eb944fe7ebb95",
-  "ERC721_COLLECTION_FACTORY_CONTRACT_ADDRESS": "0x16d8bac5b67a6b782a9081377bec413bc5bb56a6",
-  "ERC721_COLLECTION_CONTRACT_ADDRESS": "0xc3c8fb76108673f546e8c97d168af61c5573480b",
```

Comment:
I think this address is still being used in `contracts.ts`:

```ts
export const ERC721_COLLECTION_ADDRESS = config.get('ERC721_COLLECTION_CONTRACT_ADDRESS', '')
```

shouldn't we delete it from there too?

Reply:
![image](https://user-images.githubusercontent.com/24811313/182463726-f2d78c75-bdc4-4ad3-bb19-36367d9f25a5.png)